trace-dispatcher/src/Cardano/Logging/Tracer/EKG.hs

```diff
@@ -84,3 +89,9 @@ ekgTracer storeOrServer = liftIO $ do
                         Right server -> creator2 name server
             let rgsMap' = Map.insert name gauge rgsMap
             pure (rgsMap', gauge)
+
+presentPrometheusM :: [(Text, Text)] -> Text
+presentPrometheusM labels = "{" <> foldr (\(k, v) acc -> k <> "=\"" <> v <>
+                                    (if k == fst (last labels)
+                                      then "\" " <> acc
+                                      else "\", " <> acc)) "" labels <> "} 1"
```

Comment:
I find that a little hard to approach, and I think one can avoid checking being the last key in the list. Can we use something more expressive, like e.g.:
```
presentPrometheusM =
  label . map pair
  where
    label pairs = "{" <> intercalate "," pairs <> "} 1"
    pair (k, v) = k <> "=\"" <> v <> "\""
```